project/src/pages/not-found-page/not-found-page.module.css

```
@@ -0,0 +1,18 @@
+.NotFound {
```

А почему с большой?)
```suggestion
.notFound {
```